fix: don't overwrite "Show hidden files" setting in Linux/GNOME (#46926)
electron/electron: docs/api/dialog.md

    displayed in front of the filename text field.
  * `showsTagField` boolean (optional) _macOS_ - Show the tags input box, defaults to `true`.
  * `properties` string[]&#32;(optional)

>     * `showHiddenFiles` - Show hidden files in dialog. This property is not supported on Linux/GTK.

Reviewer: ```suggestion
    * `showHiddenFiles` - Show hidden files in dialog.  Deprecated on Linux. _macOS_ _Windows_ _Deprecated_
```
